### ISSUE TITLE
Transcribing to Parchment from a book of multiple pages creates second empty page

### ISSUE THREAD

**Issue by T2MWB** [anegostudios/VintageStory-Issues#2978]
### Game Version

1.18.7 (Stable)

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

When pressing "Transcribe" from a signed book with two or more pages (written on or filled with newlines), it creates a parchment that has the transcribed page but also a second page containing nothing.

The required fix is that since the transcribed parchment acts as one page, it should only have one page, maybe with no page indicator at all

Along with that I want to suggest that the page arrows in the book UI create a new page automatically when pressed at the end of the book, so that new pages before the end of the page dont have to be created by filling it with newlines

### How to reproduce

1. Take a quill in your offhand, open a book and write anything on several pages
2. Sign it with or without saving first
3. Open signed book with Parchment in inventory and press "Transcribe" on any but the last page
4. Look at the resulting transcribed parchment, which should have a second empty page (or only one if you transcribed the last page)

### Screenshots

First test: Written page with writing on the next page
First page
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/27a6effc-fcce-47a5-a39f-a7b8ad2fabe5)
Second page
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/f5b2fb4a-5065-422f-9688-9e27bc31865d)

Second test: Written page with a blank next page
First page
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/95a9be24-b56c-4560-ba69-254e4e46fe79)
Second page (same as first test so I'm copying it here)
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/f5b2fb4a-5065-422f-9688-9e27bc31865d)

Third test: blank page with a blank next page
First page
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/8eed85de-2882-45d2-a3e0-20e6328954e4)
Second page (same as first test so I'm copying it here)
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/f5b2fb4a-5065-422f-9688-9e27bc31865d)

Control test: last page to see how it should work, blank due to the order of the tests but it can be written to and will display the text
Only page
![grafik](https://github.com/anegostudios/VintageStory-Issues/assets/117268557/88469ad1-5ce9-4f5b-860a-f6a5a02f0421)

### Logs

```Log```

**Comment by ItzASecret** [comment]
yeah this bothers me a little bit because I wrote a book on the TOPS server and planned to duplicate and distribute it, but I can't transcribe it to another book, only parchment, and transcribing parchment only transcribes one page, which it should at least do 2 (front and back)

This means theres virtually no way to distribute my 10 page book other than manually re-pasting it. Unfortunate lol
